doc/src/odin/install.md

```diff
@@ -1 +1,79 @@
-## Install
\ No newline at end of file
+## Install
+
+### Prerequisites
+
+1.  ctags
+2.  bison
+3.  flex
+4.  gcc 5.x
+5.  cmake 2.8.12 (minimum version)
```

Comment:
I do believe this as changed version, https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/ODIN_II/CMakeLists.txt#L1